[docs] Update SAM verbiage in project volume doc (#798)
openshift/windows-machine-config-operator: docs/projected-volume.md

volume sources into the same directory in the container. With the [proposal for file permission handling in projected
service account volume](https://github.com/kubernetes/enhancements/pull/1598) enhancement, the projected files have
the correct permissions set including container user ownership for Linux pods when `RunAsUser` is set in the Pod 
`SecurityContext`. In Windows pods that have a projected volume and `RunAsUsername` set in the Pod `SecurityContext`,
the ownership is not enforced due to the way user accounts are managed in Windows. Windows stores and manages local

> user and group accounts in a database file called Security Account Manager (SAM). Each container maintains its own
> instance of the SAM database, to which the host has no visibility into while the container is running. Windows

Reviewer: ```suggestion
user and group accounts in a database file called Security Account Manager (SAM). Each container maintains its own
instance of the SAM database; the host machine has no visibility into the SAM database while the container is running. Windows
```
This sentence sounded awkward to me. Does this suggestion mean the same thing?

Author: This could imply the SAM database on the host. So it would be safer to leave this as is.

containers are designed to run the user mode portion of the OS in isolation from the host, hence the maintenance of a

> virtual SAM database. As a result, the kublet running on the host does not have the ability to dynamically configure

Reviewer 2: kublet -> kubelet

> host file ownership for virtualized container accounts. It is recommended that if files on the host machine are to be

Reviewer: Wording nit; perhaps this would be more concise?

```
If files on the host machine are to be shared with the container, they should be placed into
their own volume mount outside of the `C:\` drive.
```

Author: Fixed

> shared with the container then they should be placed into their own volume mount outside of `C:\`.

Reviewer 2: Is this implying that there should be a separate letter drive for each set of host files needed to be shared with different containers?

Author: Looks like it. This is coming directly from Microsoft.
